utilities/io_tools/output_tools.py

```diff
@@ -58,6 +58,10 @@ def write_analysis_output(results, outfile, args, update_name=True):
         to_append.append(args.theoryCorr[0]+"Corr")
     if args.maxFiles is not None:
         to_append.append(f"maxFiles_{args.maxFiles}".replace("-","m"))
+    if args.pdfs and args.pdfs != ["msht20"]:
+        to_append.append(args.pdfs[0])
+    if args.ptqVgen:
```

Comment:
if hasattr(args, "ptqVgen") and args.ptqVgen:

Reply:
Done